docs/admin/installation/hardware.rst

```diff
@@ -455,6 +455,27 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
+.. _nuc14_recommendation:
+
+14th-gen NUC
+~~~~~~~~~~~~
+We have tested and can recommend the `ASUS NUC14RVH <https://www.asus.com/us/displays-desktops/nucs/nuc-mini-pcs/asus-nuc-14-pro/>`__.
+It provides two M.2 SSD storage options: a 22x80 port for an NVMe drive, and a
+22x42 port for another NVMe drive. It also has a 2.5 inch drive bay for a SATA hard
```

Comment:
```suggestion
It provides both 22x80 and 22x42 M.2 ports for NVMe SSD storage, as well as a 2.5 inch drive bay for a SATA hard
```

Just a small suggestion to improve flow/concision - please feel free to take it if you like it, or leave it if you don't